pennylane/ops/__init__.py

```diff
@@ -56,13 +56,16 @@ class Identity(CVObservable, Operation):
     simulators should always be equal to 1.
     """
     num_wires = 1
-    num_params = 0
     par_domain = None
     grad_method = None
 
     ev_order = 1
     eigvals = np.array([1, 1])
 
+    @property
+    def num_params(self):
+        return 0
```

Comment:
out of curiosity, why does this need to change? isn't `num_params = 0`, at the class level, basically identical to the `@property`?

Reply:
Hmm, you may be right. I was trying to do it by the book and overwrite the property from the main class, so it inherits the docstring. What do you think?

Reply:
I have a preference to leaving it as is (as an inline class attribute), partly because it reduces the line count of the file and I find it easier for devs to 'parse' the contents and logic. But I can be persuaded either way!